### DISCUSSION TITLE
How do I use Json Schema Editor?

### DISCUSSION THREAD

**Discussion by Jefflai0315** [Budibase/budibase#4446] [Help] [answered]
I have this JSON type field in my table.
I want to create a form on my design screen that users can keep adding new items. After the user has added multiple items, I want to generate a list that stores these items. Then format and add the list to my JSON type field in my table.

Is that possible?

![image](https://user-images.githubusercontent.com/79297365/153704156-97290c2a-8d12-49dd-9fd3-0735b61df091.png)

**Comment by mjashanks** [comment, marked as answer]
So the JSON editor that you screenshotted is there so you can provide an example structure of the JSON. Just paste in an example JSON object.

Once you do this, it means that Budibase can parse this and make the properties available for Binding, and allow your arrays to be used by `Data Provider` components.

If you're asking how to save JSON, then it's don through the `Save Row` action. In my example below, the `Questionnaire` column is a json type:

![image](https://user-images.githubusercontent.com/3524181/153835053-10d61c53-14ba-4e09-b859-d72bdbe27426.png)

I then javascript to return a value for this (which in this case is simply assigning to a value from state):

![image](https://user-images.githubusercontent.com/3524181/153835199-0bd8b538-ad62-4fff-8a6c-04bc40229ec1.png)

**Comment by Jefflai0315** [comment]
Thanks :) got it
